### PR TITLE
Update release notes [skip ci]

### PR DESCRIPTION
**Pull request by devbww**
I have something for every PR, so please note anything that should be removed.

---
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/3038)

### CODE REVIEW

File: google/cloud/README.md [hunk 1]
```diff
@@ -20,7 +20,19 @@ Documentation for the common utilities is available [online][doxygen-link].
...
 ## Release Notes
 
-### v0.11.x - TBD
+### v0.12.x - TBD
+
+### v0.11.x - 2019-09
+
+* feat: Use macros for compiler id and version (#2937)
+* fix: Fix bazel build on windows. (#2940)
+* chore: disabled google-runtime-int (#2958)
```
Comment (coryan) [inline google/cloud/README.md]: I would remove that, it was just a .clang-tidy config

File: google/cloud/README.md [hunk 2]
```diff
@@ -20,7 +20,19 @@ Documentation for the common utilities is available [online][doxygen-link].
...
+* chore: Keep release tags in master branch. (#2963)
+* cleanup: Use only find_package to find dependencies. (#2967)
+* chore: Configure tsan and msan builds. (#3011)
```
Comment (coryan) [inline google/cloud/README.md]: We can probably remove that too, but up to you.

File: google/cloud/storage/README.md [hunk 3]
```diff
@@ -44,7 +44,38 @@ Apache 2.0; see [`LICENSE`](../../../LICENSE) for details.
...
 
-### v1.4.x - TBD
+### v1.5.x - TBD
+
+### v1.4.x - 2019-09
+
+* feat: Random CRC and MD5 in storage throughput benchmark (#2943)
+* feat: Make GCS throughput benchmark record progress. (#2944)
+* feat: Increase download and upload buffers. (#2945)
+* feat: Increase the threshold fur using resumable uploads (#2946)
```
Comment (coryan) [inline google/cloud/storage/README.md]: typo: fur -> for

File: google/cloud/storage/README.md [hunk 4]
```diff
@@ -44,7 +44,38 @@ Apache 2.0; see [`LICENSE`](../../../LICENSE) for details.
...
+* cleanup: Don't include nljson.h from public oauth2 headers. (#2954)
+* bug: Handle CURLE_{RECV,SEND}_ERROR as StatusCode::kUnavailable. (#2965)
+* cleanup: Use ObjectMetadata in ResumableUploadResponse. (#2969)
+* feat: Make integration tests client configurable. (#2970)
```
Comment (coryan) [inline google/cloud/storage/README.md]: Cloud remove this, but probably does not matter.

File: google/cloud/storage/README.md [hunk 5]
```diff
@@ -44,7 +44,38 @@ Apache 2.0; see [`LICENSE`](../../../LICENSE) for details.
...
+* bug: do not disable hashes when Disable{MD5,Crc32c} are set to false (#2979)
+* feat: Improve ObjectWriteStreambuf by replacing O(n^2) code. (#2989)
+* chore: Workaround gcc-8.2 build failures. (#2991)
```
Comment (coryan) [inline google/cloud/storage/README.md]: Can remove this.

File: google/cloud/storage/README.md [hunk 6]
```diff
@@ -44,7 +44,38 @@ Apache 2.0; see [`LICENSE`](../../../LICENSE) for details.
...
+* cleanup: use existing function to generate data. (#2992)
+* feat: allow applications to timeout stalled downloads. (#2993)
+* fix: Actually enable the error injection test. (#2995)
+* cleanup: Add missing TODO comment. (#2996)
```
Comment (coryan) [inline google/cloud/storage/README.md]: Probably remove this?

File: google/cloud/storage/README.md [hunk 7]
```diff
@@ -44,7 +44,38 @@ Apache 2.0; see [`LICENSE`](../../../LICENSE) for details.
...
+* cleanup: Factor out payload creation from ServiceAccountCredentials:: (#2997)
+* chore: Expose `FakeClock` for general use (#3000)
```
Comment (coryan) [inline google/cloud/storage/README.md]: Probably can remove this.

File: google/cloud/storage/README.md [hunk 8]
```diff
@@ -44,7 +44,38 @@ Apache 2.0; see [`LICENSE`](../../../LICENSE) for details.
...
+* cleanup: Add additional testing for credential helpers. (#3004)
+* fix: return proper error code from upload metadata (#3005)
+* bug: Fix the initial backoff interval. (#3007)
+* fix: eliminate a race condition from retry loop (#3013)
+* fix: add more testing to increase coverage. (#3020)
```
Comment (coryan) [inline google/cloud/storage/README.md]: Probably can remove this

File: google/cloud/storage/README.md [hunk 9]
```diff
@@ -44,7 +44,38 @@ Apache 2.0; see [`LICENSE`](../../../LICENSE) for details.
...
+* bug: Unit tests are too slow. (#3021)
+* fix: don't throw on expired retry policies (#3023)
+* bug: ReadObject() retries only once (#3028)
+* bug: Fix a SIGSEGV in test. (#3031)
```
Comment (coryan) [inline google/cloud/storage/README.md]: Probably can remove this.

File: google/cloud/README.md (hunk 1 above)
Comment (devbww) [reply google/cloud/README.md]: Done.

File: google/cloud/README.md (hunk 2 above)
Comment (devbww) [reply google/cloud/README.md]: Done.

File: google/cloud/storage/README.md (hunk 3 above)
Comment (devbww) [reply google/cloud/storage/README.md]: Done.

File: google/cloud/storage/README.md (hunk 4 above)
Comment (devbww) [reply google/cloud/storage/README.md]: Done.

File: google/cloud/storage/README.md (hunk 5 above)
Comment (devbww) [reply google/cloud/storage/README.md]: Done.

File: google/cloud/storage/README.md (hunk 6 above)
Comment (devbww) [reply google/cloud/storage/README.md]: Done.

File: google/cloud/storage/README.md (hunk 7 above)
Comment (devbww) [reply google/cloud/storage/README.md]: Done.

File: google/cloud/storage/README.md (hunk 8 above)
Comment (devbww) [reply google/cloud/storage/README.md]: Done.

File: google/cloud/storage/README.md (hunk 9 above)
Comment (devbww) [reply google/cloud/storage/README.md]: Done.

**Pull request merged**